tycho/template/pod.yaml

```diff
@@ -59,8 +59,14 @@ spec:
         cpu: {{ system.init_cpus }}
     command: [ 'sh', '-c' ]
     args:
+      # ToDo: The chown command doesn't seem to work all the time?  owner:group can end up being 65534:65534.
+      # Maybe it is only when the path is used as a subpath and mounted.
       - mkdir -p {{ system.parent_dir }}/{{ system.subpath_dir }} &&
+        chown {{ system.security_context.run_as_user }}:{{ system.security_context.run_as_group }} {{ system.parent_dir }}/{{ system.subpath_dir }} &&
```

Comment:
I think for this to work the TYCHO_APPS_RUN_AS_USER and TYCHO_APPS_RUN_AS_GROUP always need to be set.

Reply:
I think there should be a default.